sdk/core/Azure.Core/src/AsyncPageable.cs

```diff
@@ -79,8 +79,8 @@ public abstract class AsyncPageable<T> : IAsyncEnumerable<T> where T : notnull
         /// begin paging from the beginning.
         /// </param>
         /// <param name="pageSizeHint">
-        /// The size of <see cref="Page{T}"/>s that should be requested (from
-        /// service operations that support it).
+        /// The number of items per <see cref="Page{T}"/> that should be requested (from
```

Comment:
Can we ever get more, or will this always be the maximum?

Reply:
In many services, this parameter is completely ignored.